### ISSUE TITLE
Practical Deep Learning course, November 2022

### ISSUE THREAD

**Issue by mvsjober** [csc-training/intro-to-dl#8]
Recurring TODOs:
- [x] Reserve Dogmi (Mats)
- [x] Fill course form (Mats)
- [x] Create course project (Mats)
- [x] Training accounts (Mats)
- [x] GPU reservation (Mats)
- [x] Eventilla page (Marina)
- [x] PRACE page (Marina)
- [x] Notebooks course request (Mats)
- [x] Update/review lectures (Mats)
- [x] Update/review lectures (Markus)
- [x] Make tinyurl
- [x] Test Notebooks + exercises
- [x] Test Puhti exercises
- [x] Fix project in Puhti exercises (5299 => 6678) (Markus)
- [x] Test with training accounts
- [x] Test day1 with Google colab
- [x] Create Zoom room (Mats)
- [x] HackMD (or replacement) pages
- [x] Copy data from old to new project in Puhti (5299 => 6678) (Markus)

TODOs from last time's comments:
- [x] Schedule 15 less for L3 and 15 more for L4
- [x] some errors in exercise 2, with references to history instead of history_ex1 etc
- [x] Write down somewhere to mention notebook lifetime and how to save/transfer files from Notebooks
- [x] Re-use, finetuning should be explained better. I should use the image in the lecture better. Also the code should be simplified (remove the non-vgg models etc).
- [ ] Take a look at PyTorch vs Keras accuracy differences in CNN/RNN 20ng exercises.
- [x] L1 was too long again. Maybe the SGD stuff should be moved to L2?
- [ ] Move any new git commits to mls

After registration has closed (recurring):
- [x] Request Notebooks training accounts (Mats)
- [x] Send email to Marina with: zoom-link, materials link (Mats)

During course:
- [x] Remember to record lectures!!

After course:
- [x] Post lecture recordings
- [x] Send attendance info to Marina (→ certificates)

Specific to this instance:
- [x] Set up HedgeDoc to use instead of HackMD (if all goes well)
- [x] HedgeDoc sub pages for discussion, day 1 exercises, day 2 exercises, passwords
- [ ] Consider using NVME in all scripts?
- [ ] Convolutions lecture update? ConvNeXt et al?
- [x] Embeddings lecture update? Sentence embeddings, image embeddings, CLIP et al?
- [x] Transformers lecture update?
- [x] Remove mobilenet option, try efficientnet instead in exercises
- [x] Ask about/reserve wireless microphones for Dogmi (Mikael K.)
- [ ] CVSLogger in exercises

**Comment by mvsjober** [comment]
Eventilla: https://ssl.eventilla.com/event/aqew6
PRACE: https://events.prace-ri.eu/event/1437/

**Comment by mvsjober** [comment]
Course project is 2006678

**Comment by mvsjober** [comment]
TinyUrl: https://tinyurl.com/csc-pdl-2022-11

**Comment by mvsjober** [comment]
HackMD: https://hackmd.io/0-rOHhIqRVCcxagmiy_Mug

**Comment by mvsjober** [comment]
Notebooks workspace join code: `pra-4l8ubd99`

**Comment by mvsjober** [comment]
Day 1 morning session was running over time, so that the exercise before lunch was basically squeezed into 15 minutes after lunch.

**Comment by mvsjober** [comment]
We kept forgetting to turn on recording on day2. How to fix this? Post-it notes? Note on first slide "START RECORDING" ? :-)

**Comment by mvsjober** [comment]
When to show CNN applications slides? Maybe on day 1 after all?

**Comment by MatiasJJ** [comment]
- SLURM output-files could be named like the input scripts so it's easier to know which results to read

**Comment by mvsjober** [comment]
Day 2 morning has too much talk by me, and then there essentially 10 mins for actually doing exercises. Maybe skip the frameworks part in the beginning?

**Comment by MatiasJJ** [comment]
Tables in exercises: each model could have their own so it would be easier to compare
